doc/sphinx/source/env_variables.rst

```diff
@@ -33,6 +33,11 @@ pocl.
  CL_INVALID_VALUE. If clock_gettime is available, messages
  will include a timestamp.
 
+ The old way (setting POCL_DEBUG to 1) has been updated to support categories.
```

Comment:
1) POCL_DEBUG writes to stderr, not stdout (this might be useful sometimes), is there a way to redirect stderr only to grep ?

2) I've already written the code ;)

3) "program | grep" means program's stderr/stdout is a pipe, not a tty -> no colors. Colors make it (for me) a lot easier to find what i need in the debug output...

But you can use grep on prefixes after this commit, though the non-tty output is slightly different.